### PR TITLE
Added prefix for the api urls.

### PR DESCRIPTION
**Pull request by topless**

### CODE REVIEW

**Comment by gmist** [comment]
+1

**Comment by lipis** [comment]
Supporting other versions will be quite a challenge though with this model to be future proof... :) 

What about having something like that in the main.py:

```python
api_v1 = helpers.Api(app, prefix='/api/v1')
```

and then importing that thing from main inside the `api.v1` modules..

doing that it will be easier in the future to just create new `v2`, `v3` modules and change only the import and in the main.py it will look like like:

```python
api_v1 = helpers.Api(app, prefix='/api/v1')
api_v2 = helpers.Api(app, prefix='/api/v2')
api_v3 = helpers.Api(app, prefix='/api/v3')
...
```

WDYT?

File: main/main.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 import task
 
 from api import helpers
-api = helpers.Api(app)
+api = helpers.Api(app, prefix='/api/v1')
```
Comment (lipis) [inline main/main.py]:
```python
api_v1 = helpers.Api(app, prefix='/api/v1')
```

**Comment by topless** [comment]
Well in my head there are very few apis that went to version 2. Maybe `api1` or `api_1` or fuck it and leave it as api cause its 3 chars and really sweet. What @gmist @mdxs @joernhees you guys think?

**Comment by lipis** [comment]
https://developers.google.com/analytics/devguides/collection/android/v4/
https://developers.google.com/maps/documentation/javascript/

:)  APIs are going up in general :P

**Comment by topless** [comment]
I am rebasing it on top of the master and I was looking around for the `import api` in order to do the refactor. Considering your suggestion
`api_v1 = helpers.Api(app, prefix='/api/v1')`
`import api_v1.v1`
You see my point, so what if we refactored the api directory into `api_v1` and remove the `v1` so we can escape the v1.v1 import ugliness.

File: main/api/v1/config.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 from main import api
```
Comment (lipis) [inline main/api/v1/config.py]: this will become 
```python
from main import api_v1
```

**Comment by topless** [comment]
I think I got confused but I was referring to the import in main.py line 27.

**Comment by lipis** [comment]
yep.. that will remain as is :D

**Comment by topless** [comment]
I got confused because the variable `api` and the module `api` share the same name .. although they are different things :)

**Pull request merged**